tests/ServerlessAutoSwagger.test.ts

```diff
@@ -820,6 +820,7 @@ describe('ServerlessAutoSwagger', () => {
           {
             basePath: '/bp',
             host: 'some-host',
+            title: 'Serverless service name',
```

Comment:
```suggestion
            title: 'My API Title',
```
and then line 836 should change from

```typescript
info: expect.any(Object)
```
to
```typescript
info: { title: 'My API Title', version: '1' },
```